features/image-transformations/resize-crop-and-other-transformations.md

```diff
@@ -747,17 +747,17 @@ This is good for creating profile pictures.
 {% endtab %}
 {% endtabs %}
 
-### Background color - (bg)
+### Background - (bg)
 
-It is used to specify the background color that can be used along with some cropping strategies while resizing an image.
+It is used to specify the background that can be used along with some cropping strategies while resizing an image. Background can be any solid color or blurred image.
```

Comment:
"This feature allows you to define the background when resizing an image, and it works seamlessly with various cropping strategies. You can choose either a solid color or a blurred version of the same image as the background."

Bring the limitation message up here.